pixels-turbo/pixels-worker-common/src/main/java/io/pixelsdb/pixels/worker/common/WorkerCommon.java

```diff
@@ -80,7 +79,7 @@ public static void initStorage(StorageInfo storageInfo)
             }
             else if (WorkerCommon.minio == null && storageInfo.getScheme() == Storage.Scheme.minio)
             {
-                ConfigMinio(storageInfo.getEndpoint(), storageInfo.getAccessKey(), storageInfo.getSecretKey());
+//                ConfigMinio(storageInfo.getEndpoint(), storageInfo.getAccessKey(), storageInfo.getSecretKey());
```

Comment:
This line is necessary.

Reply:
let me change the logic of minio and test it